src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/FinanceTrackerParserTest.java

```diff
@@ -245,31 +245,55 @@ public void parseCommand_addBookmarkExpenseWhenAnalyticsTab() throws Exception {
     @Test
     public void parseCommand_clearWhenOverviewTab() throws Exception {
         assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD, overviewUiStateStub) instanceof ClearCommand);
-        assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD + " 3",
+        assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD,
                 overviewUiStateStub) instanceof ClearCommand);
```

Comment:
Remove duplicate code. Same for other places within this class.

Reply:
Could you point out where code is duplicated? I don't see any instances of code duplication.

Reply:
I believe it's the immediately adjacent line?

https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/281/files/3003e94a7bea54c4ad94382624a47dacd9ca0ba2#diff-2ef7d0df41f8470e31994dbff5f1604d90853f623e33985441ddf125f009901eR261-R263

![image](https://user-images.githubusercontent.com/53135010/98197980-e853e380-1f62-11eb-86cb-976246b9ce8a.png)

Reply:
Ah okay, I see now!